docs/reference-architectures/index.rst

```diff
@@ -0,0 +1,14 @@
+=======================
+Reference Architectures
+=======================
+
+This section of the documentation covers reference architectures involving
+CrateDB for various use-cases.
+
+.. rubric:: Azure
+
+.. toctree::
+   :maxdepth: 1
+
+   cratedb-azure-iot
```

Comment:
is it possible to have a "teaser" text in this section?
This might be helpful here.

Reply:
@mechanomi Is this something that could be done in sphinx? Or would we have to ape it with a list of links with descriptions?

Reply:
Shouldn't be too hard to write manually anyway